### PR TITLE
feat: bump gradle api to 7.6.2

### PR DESCRIPTION
**Pull request by manusa**
## Description

feat: bump gradle api to 7.6.2

Relates to
- #2299
- https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/9839
  > @manusa
  > This excerpt from the [handbook](https://www.eclipse.org/projects/handbook/#ip-third-party) seems applicable, since Gradle is a build tool and can be considered the platform that will eventually execute our plugin.
  
  > @rahulmohang
  > Hi [@mnuri](https://gitlab.eclipse.org/mnuri),
  >
  > As of now, we think that this assessment is right. 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [x] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,13 +89,15 @@
   <properties>
     <jdk.version>1.8</jdk.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <skipTests>false</skipTests>
```
Comment (rohanKanojia) [inline pom.xml:92]: Why is this change required?

**Review by rohanKanojia — APPROVED** [review]

**Comment by manusa** [comment]
According to the conversation in https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/9839, there's no need to vet the **provided** Gradle dependencies. In this case the dependencies are provided by the user at runtime while running the Gradle build tool which can be considered as the runtime platform.

```
maven/mavencentral/com.marcnuri.plugins/gradle-api-maven-plugin/0.0.2, Apache-2.0, approved, clearlydefined
```
